src/closed.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="21px" height="21px" viewBox="0 0 21 21" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
We should change `times.svg` instead of creating a new file

Reply:
I'd rather delete than one, what do you think?

Reply:
I think `times` is a more generic name

Reply:
Good point. Changing it.